src/quo2/foundations/colors.cljs

```diff
@@ -122,6 +122,8 @@
 (def customization
   {:primary   {50 primary-50  ;; User can also use primary color as customisation color
                60 primary-60}
+   :danger    {50 danger-50
+               60 danger-60}
```

Comment:
we don't allow users to set `:danger` as their profile or community theme. So it shouldn't be in customization colors.

Reply:
Done, implemented as discussed (separated from customizations)